packages/paste-website/src/pages/components/tooltip/index.mdx

```diff
@@ -0,0 +1,222 @@
+package: '@twilio-paste/tooltip'
+description: An accessible and styled tooltip component.
+slug: /components/tooltip/
+---
+
+import {graphql} from 'gatsby';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {Text} from '@twilio-paste/text';
+import Changelog from '@twilio-paste/tooltip/CHANGELOG.md';
+import {Tooltip} from '@twilio-paste/tooltip';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {SidebarCategoryRoutes} from '../../../constants';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/tooltip"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/tooltip/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/tooltip"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-tooltip--default"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+<Callout>
+  <CalloutTitle>Using alpha components in your product</CalloutTitle>
+  <CalloutText>
+    Alpha components are ready for you to use in production. Components such as Tooltip are likely to not see any API
+    changes. This is only an alpha due to the lack of documentation and a matching Sketch symbol. Feel free to share
+    your experience using this component by{' '}
+    <Anchor href="https://github.com/twilio-labs/paste/discussions/new">starting a discussion on GitHub</Anchor>. We are
+    also looking for a contributor to symbolize this component in Sketch. Interested? Reach out!
+  </CalloutText>
+</Callout>
+
+## Guidelines
+
+### About Tooltip
+
+The Tooltip component is a popup that displays text related to an element when the element is focused via
+the keyboard or on mouse hover. Tooltip follows the
+[WAI-ARIA Tooltip Pattern](https://www.w3.org/TR/wai-aria-practices/#tooltip).
+
+## Examples
+
+### Basic Tooltip
+
+<LivePreview
+  scope={{
+    Tooltip,
+    Button,
+  }}
+>
+  {`<Tooltip text="Black Lives Matter. We stand with the Black community.">
+  <Button variant="primary">✊ Action</Button>
+</Tooltip>`}
+</LivePreview>
+
+### Controlling placement
+
+The placement of the tooltip is configurable via the `placement` prop. The available placement options are
+available in the [props description here](#tooltip). By default the tooltip is positioned automatically so
+you don't have to worry about it going off the page, or being covered by another element.
```

Comment:
Might be worth leaving a note about placement if setting it forces the tooltip off-screen, what happens? Or rewording " By default the tooltip is positioned automatically" to something like "no matter the placement, the tooltip will always adjust to remain in the screen"?

Reply:
Added some more text.